articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
+  ]
+}
+```
 
-```text
-${manage_url}/#/applications/YOUR_API_ID/settings
+<div class="alert alert-info">
+  You can get the curl command for each endpoint from the Management API v2 Explorer. Go to the endpoint you want to call, and click the <em>get curl command</em> link at the <em>Test this endpoint</em> section.
+</div>
+
+That's it! You are done!
+
+### Sample Implementation: Python
+
+This python script gets a Management API v2 access token, uses it to call the [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint, and prints the response in the console:
+
+```python
+def main():
+  import json, urllib, urllib2
+
+  # Configuration Values
+  DOMAIN = "${account.namespace}"
+  AUDIENCE = "https://${account.namespace}/api/v2/"
+  CLIENT_ID = "${account.clientId}"
+  CLIENT_SECRET = "${account.clientSecret}"
+  GRANT_TYPE = "client_credentials" # OAuth 2.0 flow to use
+
+  # Get an access token from Auth0
+  base_url = "https://{domain}".format(domain=DOMAIN)
+  data = urllib.urlencode([('client_id', CLIENT_ID),
+                          ('client_secret', CLIENT_SECRET),
+                          ('audience', AUDIENCE),
+                          ('grant_type', GRANT_TYPE)])
+  req = urllib2.Request(base_url + "/oauth/token", data)
+  response = urllib2.urlopen(req)
+  oauth = json.loads(response.read())
+  access_token = oauth['access_token']
+
+  # Get all Clients using the token
+  req = urllib2.Request(base_url + "/api/v2/clients")
+  req.add_header('Authorization', 'Bearer ' + access_token)
+  req.add_header('Content-Type', 'application/json')
+
+  try:
+    response = urllib2.urlopen(req)
+    res = json.loads(response.read())
+    print res
+  except urllib2.HTTPError, e:
+    print 'HTTPError = ' + str(e.code) + ' ' + str(e.reason)
+  except urllib2.URLError, e:
+    print 'URLError = ' + str(e.reason)
+  except urllib2.HTTPException, e:
+    print 'HTTPException'
+  except Exception:
+    print 'Generic Exception'
+
+# Standard boilerplate to call the main() function.
+if __name__ == '__main__':
+  main()
 ```
 
-You can then change the `Client Secret` in that page to change your Management APIv2 secret.
+## Frequently Asked Questions
+
+__How long is the token valid for?__</br>
+The Management APIv2 token has a validity of __24 hours__. After that the token will expire and you will have to get a new one.
+
+__I hate this! The Management APIv2 token used to never expire, why would you change that?__</br>
```

Comment:
Perhaps we should remove the hate this part? and put something like "why was it changed?" instead of "why would you" to make the change seem more natural and avoid telling users that it has generated some discomfort